.github/workflows/cibuildwheels.yml

```diff
@@ -54,7 +58,7 @@ jobs:
           python -m pip install -r requirements-tests.txt
 
       - name: Build wheels (Windows x86)
-        if: ${{ runner.os == 'Windows' && matrix.arch == 'x86'}}
+        if: ${{ runner.os == 'Windows' }}
```

Comment:
Why you are removing the `matrix.arch == 'x86'` condition?  Is that related with `auto`?

Reply:
Since `x86` and `amd64` are replaced with `auto`, this condition is removed. Also, `CIBW_BUILD` is provided in the `env` which will build the required wheel.

Reply:
Since `x86` and `amd64` are replaced with `auto`, this condition is removed. Also, `CIBW_BUILD` is provided in the `env` which will build the required wheel.

Reply:
Understood.  So I'd say that the section below ('Build wheels (Windows / amd64)') is redundant.

Reply:
If I understand this correctly, this also means that this should be producing wheels for `aarch64` arch too.

Reply:
No, It produces only for the supported architectures of a build machine.

Reply:
> Understood. So I'd say that the section below ('Build wheels (Windows / amd64)') is redundant.

Yes, this can be removed.

Reply:
Thanks!